test/integration/file-uploads/request-body.test.js

```diff
@@ -645,4 +650,65 @@ describe('file upload in request body', function () {
                 .to.equal('Binary file load error: file resolver interface mismatch');
         });
     });
+
+    (IS_BROWSER ? describe.skip : describe)('should upload large files correctly', function () {
+        var testUploadFile = 'test/fixtures/upload-file-large.json';
+
+        afterEach(function () {
+            sh.rm('-rf', testUploadFile);
+        });
+
+        // eslint-disable-next-line mocha/no-sibling-hooks
+        before(function (done) {
+            this.enableTimeouts(false);
+            if (IS_DARWIN) {
```

Comment:
Nitpick: We can separate individual blocks of code by a newline. e.g. in this case, the `if` block can start after a newline and a newline after the last `else if`.